src/rules-engine/effects/ActionEffect.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.*
+ */
+
+const assert = require('assert');
+const fetch = require('node-fetch');
+const Effect = require('./Effect');
+const Settings = require('../../models/settings');
+
+/**
+ * An Effect which creates an action
```

Comment:
Heh, this would have been confusing if Effect was also still called Action! Good job we renamed it.

Reply:
Yep, our acting to rename had a very positive effect